pandas-stubs/core/frame.pyi

```diff
@@ -86,6 +86,7 @@ from pandas._typing import (
     IndexingInt,
     IndexLabel,
     IndexType,
+    InterpolateOptions,
```

Comment:
Maybe the `InterpolateOptions` stuff should go in a separate PR

Reply:
I agree - I naively assumed that adding `__iter__` would be simply :)